be/src/exec/vectorized/hdfs_scanner_text.cpp

```diff
@@ -98,17 +98,34 @@ Status HdfsScannerCSVReader::_fill_buffer() {
 
 Status HdfsTextScanner::do_init(RuntimeState* runtime_state, const HdfsScannerParams& scanner_params) {
     TTextFileDesc text_file_desc = _scanner_params.scan_ranges[0]->text_file_desc;
+
+    // All delimiters will not be empty.
+    // Even if the user has not set it, there will be a default value.
+    DCHECK(!text_file_desc.field_delim.empty());
+    DCHECK(!text_file_desc.line_delim.empty());
+    DCHECK(!text_file_desc.collection_delim.empty());
+    DCHECK(!text_file_desc.mapkey_delim.empty());
```

Comment:
```suggestion
    if (text_file_desc.field_delim.size() != 1) { return Status::Unsupported("xxxx"); }
    ......
```

Reply:
Added some comments to explain the behavior.